sorc/link_fv3gfs.sh

```diff
@@ -339,6 +339,15 @@ else
 fi
 #------------------------------
 
+#------------------------------
+#--link ecflow directories
+#------------------------------
+
+if [ $machine == "wcoss2" ]; then
+  cd $pwd/..
+  [[ -s ecf ]] && mv ecf ecf-old
+  $SLINK $pwd/wcoss2_ecf_cycled/ecf
+fi
```

Comment:
In light of the above comment, this is not a valid solution.

Reply:
This is the same question, please communicate in the same communication dialog box above.

Reply:
It is not the same.  
The comment above is a question about introducing a new repository and its associated issues.
Comment here is about moving/linking/duplicating existing scripts and then using them.

And comments are made where they are relevant, even if related.

Reply:
Agreed to modify the solution above. This is part of the same solution that will be modified.